--- superkaramba/src/Makefile.in.orig	2005-07-08 04:40:59.000000000 +0200
+++ superkaramba/src/Makefile.in	2005-08-03 12:36:45.000000000 +0200
@@ -400,6 +400,7 @@
 kde_templatesdir = @kde_templatesdir@
 kde_wallpaperdir = @kde_wallpaperdir@
 kde_widgetdir = @kde_widgetdir@
+desktopdir = @desktopdir@
 kdeinitdir = @kdeinitdir@
 libdir = @libdir@
 libexecdir = @libexecdir@
@@ -481,7 +482,7 @@
 superkaramba_LDADD = -lkio $(LIB_KDEUI) $(XMMS_LDADD) $(LIBPYTHON) $(LIBKVM) $(LIB_KNEWSTUFF)
 
 # this is where the desktop file will go
-shelldesktopdir = $(kde_appsdir)/Utilities
+shelldesktopdir = $(desktopdir)
 shelldesktop_DATA = superkaramba.desktop
 
 # this is where the shell's XML-GUI resource file goes
