--- superkaramba/src/superkaramba.desktop.orig	2005-11-22 04:03:13.000000000 +0100
+++ superkaramba/src/superkaramba.desktop	2005-12-03 23:17:47.730061696 +0100
@@ -57,5 +57,6 @@
 GenericName[xx]=xxDesktop Widgetsxx
 GenericName[zh_CN]=桌面部件
 DocPath=superkaramba/index.html
-Categories=Qt;KDE;Utility;X-KDE-Utilities-Desktop;
+Terminal=false
+Categories=Application;KDE;Amusement;Applet;
 OnlyShowIn=KDE;
