--- src/karamba.desktop.old	2005-05-26 11:19:51.000000000 +0200
+++ src/karamba.desktop		2005-05-26 11:21:28.000000000 +0200
@@ -1,7 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=SuperKaramba
+Comment=An engine for cool desktop eyecandy
+Comment[pl]=Silnik dla super wodotrysków na pulpit
 Exec=superkaramba
 Icon=clanbomber
+Terminal=false
 Type=Application
-Comment=An engine for cool desktop eyecandy.
+OnlyShowIn=KDE
+Categories=Application;KDE;Amusement;Applet;
+# vi: encoding=utf-8
--- src/superkaramba.desktop.old	2005-05-26 11:19:37.000000000 +0200
+++ src/superkaramba.desktop		2005-05-26 11:20:52.000000000 +0200
@@ -1,7 +1,12 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=SuperKaramba
+Comment=An engine for cool desktop eyecandy
+Comment[pl]=Silnik dla super wodotrysków na pulpit
 Exec=superkaramba
 Icon=clanbomber
+Terminal=false
 Type=Application
-Comment=An engine for cool desktop eyecandy.
+OnlyShowIn=KDE
+Categories=Application;KDE;Amusement;Applet;
+# vi: encoding=utf-8
