--- superkaramba-0.35/configure	2004-11-16 05:12:48.000000000 +0000
+++ superkaramba-0.35.new/configure	2004-12-17 13:51:58.999799616 +0000
@@ -25940,7 +25940,7 @@
 python_libdir=NO
 for i in $python_libdirs;
 do
-  for j in libpython$version.a;
+  for j in libpython$version.so;
   do
     echo "configure: 25945: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -25951,12 +25951,12 @@
   done
 done
 
-if test ! -r $python_libdir/libpython$version.a; then
+if test ! -r $python_libdir/libpython$version.so; then
 
 python_libdir=NO
 for i in $python_libdirs;
 do
-  for j in python$version/config/libpython$version.a;
+  for j in python$version/config/libpython$version.so;
   do
     echo "configure: 25961: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -25968,7 +25968,7 @@
 done
 
   python_libdir=$python_libdir/python$version/config
-  if test ! -r $python_libdir/libpython$version.a; then
+  if test ! -r $python_libdir/libpython$version.so; then
     python_libdir=no
   fi
 fi
@@ -25979,12 +25979,12 @@
   LIBPYTHON=-lpython$version
 fi
 
-python_libdirs="$ac_python_dir/lib /usr/lib /usr/local /usr/lib $kde_extra_libs"
+python_libdirs="$ac_python_dir/lib /usr/lib /usr/share /usr/local /usr/lib $kde_extra_libs"
 
 python_moddir=NO
 for i in $python_libdirs;
 do
-  for j in python$version/copy.py;
+for j in python$version/copy.pyc;
   do
     echo "configure: 25989: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -25996,7 +25996,7 @@
 done
 
 python_moddir=$python_moddir/python$version
-if test ! -r $python_moddir/copy.py; then
+if test ! -r $python_moddir/copy.pyc; then
   python_moddir=no
 fi
 
--- superkaramba-0.35/src/karamba.cpp	2004-11-06 12:04:00.000000000 +0000
+++ superkaramba-0.35.new/src/karamba.cpp	2004-12-17 13:52:38.207839088 +0000
@@ -56,7 +56,7 @@
   QTimer::singleShot(0, this, SLOT(initPythonInterface()));
 
   //Matthew Kay: set window type to "dock" (plays better with taskbar themes this way)
-  KWin::setType(winId(), NET::Dock);
+    KWin::setType(winId(), NET::Normal);
 
 #if defined(KDE_MAKE_VERSION)
 # if KDE_VERSION >= KDE_MAKE_VERSION(3,1,9)
