--- superkaramba/src/Makefile.in.orig	2005-11-22 04:04:41.000000000 +0100
+++ superkaramba/src/Makefile.in	2005-12-03 23:24:13.678388632 +0100
@@ -387,6 +387,7 @@
 kde_templatesdir = @kde_templatesdir@
 kde_wallpaperdir = @kde_wallpaperdir@
 kde_widgetdir = @kde_widgetdir@
+desktopdir = @desktopdir@
 kdeinitdir = @kdeinitdir@
 libdir = @libdir@
 libexecdir = @libexecdir@
@@ -467,7 +468,7 @@
 superkaramba_LDADD = -lkio $(LIB_KDEUI) $(XMMS_LIBS) $(LIBPYTHON) $(LIBKVM) $(MY_LIBKNEWSTUFF)
 
 # this is where the desktop file will go
-shelldesktopdir = $(kde_appsdir)/Utilities
+shelldesktopdir = $(desktopdir)
 shelldesktop_DATA = superkaramba.desktop
 
 # this is where the shell's XML-GUI resource file goes
