diff -urN oryg/configure popr/configure
--- oryg/configure	2003-09-30 03:39:13.000000000 +0200
+++ popr/configure	2004-04-02 19:35:13.000000000 +0200
@@ -24579,7 +24579,7 @@
 python_libdir=NO
 for i in $python_libdirs;
 do
-  for j in libpython$version.a;
+  for j in libpython$version.so;
   do
     echo "configure: 24584: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -24590,12 +24590,12 @@
   done
 done
 
-if test ! -r $python_libdir/libpython$version.a; then
+if test ! -r $python_libdir/libpython$version.so; then
 
 python_libdir=NO
 for i in $python_libdirs;
 do
-  for j in python$version/config/libpython$version.a;
+  for j in python$version/config/libpython$version.so;
   do
     echo "configure: 24600: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -24607,7 +24607,7 @@
 done
 
   python_libdir=$python_libdir/python$version/config
-  if test ! -r $python_libdir/libpython$version.a; then
+  if test ! -r $python_libdir/libpython$version.so; then
     python_libdir=no
   fi
 fi
@@ -24618,12 +24618,12 @@
   LIBPYTHON=-lpython$version
 fi
 
-python_libdirs="$ac_python_dir/lib /usr/lib /usr/local /usr/lib $kde_extra_libs"
+python_libdirs="$ac_python_dir/lib /usr/lib /usr/share /usr/local /usr/lib $kde_extra_libs"
 
 python_moddir=NO
 for i in $python_libdirs;
 do
-  for j in python$version/copy.py;
+  for j in python$version/copy.pyc;
   do
     echo "configure: 24628: $i/$j" >&5
     if test -r "$i/$j"; then
@@ -24635,7 +24635,7 @@
 done
 
 python_moddir=$python_moddir/python$version
-if test ! -r $python_moddir/copy.py; then
+if test ! -r $python_moddir/copy.pyc; then
   python_moddir=no
 fi
 
diff -urN oryg/src/karamba.cpp popr/src/karamba.cpp
--- oryg/src/karamba.cpp	2003-10-19 22:02:45.000000000 +0200
+++ popr/src/karamba.cpp	2004-03-21 23:30:38.000000000 +0100
@@ -2243,7 +2243,7 @@
 
 
     //Matthew Kay: set window type to "dock" (plays better with taskbar themes this way)
-    KWin::setType(winId(), NET::Dock);
+    KWin::setType(winId(), NET::Normal);
 
     widgetMask = 0;
     info = new NETWinInfo( qt_xdisplay(), winId(), qt_xrootwin(), NET::WMState );
